src/pages/cookie-policy/index.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import Layout from '@theme/Layout';
+import styles from 'styled-components';
+
+/**
+ * Cookie Policy
+ **/
+
+const CookiePolicyContainer = styles.div`
+  max-width: 800px;
+  margin: 0 auto;
+  padding: 20px;
+  border-radius: 5px;
+  box-shadow: 0 0 10px 0 rgba(0, 0, 0, 0.1);
+  margin-top: 20px;
+  margin-bottom: 20px;
+  font-size: 16px;
+  line-height: 1.6;
+  font-family: 'Open Sans', sans-serif;
+  text-align: justify;
+`;
+
+const Title = styles.h2`
+  margin-bottom: 20px;
+  font-weight: 700;
+  text-align: center;
+`;
+
+const SubTitle = styles.h3`
+  margin-bottom: 20px;
+  font-weight: 700;
+`;
+
+const StyledUl = styles.ul`
+  margin-bottom: 20px;
+`;
+
+const StyledLi = styles.li`
+  margin-bottom: 20px;
+`;
+
+const StyledOl = styles.ol`
+  margin-bottom: 20px;
+`;
+
+const Content = styles.div`
+  margin-bottom: 20px;
+`;
+
+/**
+ * Cookie Section
+ **/
+
+const CookieSection = ({ title, children }) => (
+  <section>
+    <SubTitle>{title}</SubTitle>
+    {children}
+  </section>
+);
+
+/**
+ * Cookie Policy
+ **/
+
+const Cookiepolicy = () => {
```

Comment:
also define `const Cookiepolicy = () => {` to `const Cookiepolicy: React.FC = () => {`